### PR TITLE
[LM-10902] Analytics

### PR DESCRIPTION
**Pull request by sidcr7-likeminds**
## Description

**Features**

- Added analytics events

**Bugs**

**What kind of change does this PR introduce?** (check at least one)

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Hotfix (fix for breaking changes)
- [x] New Feature (non-breaking change which adds functionality)
- [ ] Other, please describe:

### Author Checklist

- [x] Assignee added
- [x] Reviewer added
- [x] Description added
- [ ] Unit test cases added

### Reviewer Checklist

- [x] PR title starts with Jira ticket number prefix
- [ ] If hotfix, Add Hotfix as prefix in PR title
- [x] Label added
- [x] Commit messages follows standard format (added, changed, removed, code commented and chore)
- [x] Description about the Feature/ Bugs added

### CODE REVIEW

File: likeminds-feed-android-core/src/main/java/com/likeminds/feed/android/core/post/detail/view/LMFeedPostDetailFragment.kt [hunk 1]
```diff
@@ -1716,14 +1753,15 @@ open class LMFeedPostDetailFragment :
 
     override fun onPostShareClicked(position: Int, postViewData: LMFeedPostViewData) {
         super.onPostShareClicked(position, postViewData)
-
+        //todo: post as variable and take domain here
```
Comment (ishaan1607) [inline likeminds-feed-android-core/src/main/java/com/likeminds/feed/android/core/post/detail/view/LMFeedPostDetailFragment.kt:1756]: have we added this in next prs?

File: likeminds-feed-android-core/src/main/java/com/likeminds/feed/android/core/utils/analytics/LMFeedAnalytics.kt [hunk 2]
```diff
@@ -89,6 +91,22 @@ object LMFeedAnalytics {
         //todo add global callback
```
Comment (ishaan1607) [inline likeminds-feed-android-core/src/main/java/com/likeminds/feed/android/core/utils/analytics/LMFeedAnalytics.kt:91]: please add the callback

**Review by ishaan1607 — CHANGES_REQUESTED** [review]

File: likeminds-feed-android-core/src/main/java/com/likeminds/feed/android/core/utils/analytics/LMFeedAnalytics.kt (hunk 2 above)
Comment (sidcr7-likeminds) [reply likeminds-feed-android-core/src/main/java/com/likeminds/feed/android/core/utils/analytics/LMFeedAnalytics.kt:91]: added in further PRs

File: likeminds-feed-android-core/src/main/java/com/likeminds/feed/android/core/post/detail/view/LMFeedPostDetailFragment.kt (hunk 1 above)
Comment (sidcr7-likeminds) [reply likeminds-feed-android-core/src/main/java/com/likeminds/feed/android/core/post/detail/view/LMFeedPostDetailFragment.kt:1756]: added in further PRs

**Review by ishaan1607 — APPROVED** [review]

**Pull request merged**